assignment.hs

```diff
@@ -78,8 +78,7 @@ reciprocal x = . . .
 --
 --  Uncomment (5) to verify
 
--- your code
-
+sumOfFirst x l = sum(take x l)
```

Comment:
We normally leave a space between a function name and the open parenthesis, just to remind ourselves that the "()" is not syntax for the parameters of a function.